.github/workflows/ci.yml

```diff
@@ -29,12 +29,19 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
       - name: Build and test artifacts
         run: ./gradlew build
-      - name: Deploy artifacts
-        run: ./gradlew build artifactoryPublish -x test
+        env:
+          USER: 'uw-it-edm-bot'
```

Comment:
Consider providing the user as a secret to follow conventions in other EDM Repos. This would allow changing the user & password without having to commit a change

Reply:
changed it to "USER: ${{ secrets.EDM_GITHUB_USER }}".
shortened "secrets.EDM_GITHUB_ACCESS_PAT" to "secrets.EDM_GITHUB_PAT"